src/main/daml/Daml/Finance/Interface/Types/Common.daml

```diff
@@ -52,6 +53,17 @@ data InstrumentKey = InstrumentKey
       -- ^ A textual instrument version.
   deriving (Eq, Ord, Show)
 
+-- | A unique key for Instructions.
+data InstructionKey = InstructionKey
```

Comment:
This type definition can probably live in D.F.I.Settlement?

Reply:
agree